src/Console/ConfigurationResolver.php

```diff
@@ -702,16 +703,106 @@ private function validateRules(array $rules): void
         );
 
         if (\count($unknownFixers)) {
+            
+            $renamedRules = [
+            	'blank_line_before_return' => [
+            		'new_name' => 'blank_line_before_statement',
+            		'note' => ['statements' => ['return']]
+            	],
+                'final_static_access' =>
+            	[
+            		'new_name' => 'self_static_accessor',
+            		'note' => []
+            	],
+                'hash_to_slash_comment' =>
+            	[
+            		'new_name' => 'single_line_comment_style',
+            		'note' => ['comment_types' => ['hash']]    
+            	],
+                'lowercase_constants' =>
+            	[
+            		'new_name' => 'constant_case',
+            		'note' => ['case' => 'lower']
+            	],
+                'no_extra_consecutive_blank_lines' =>
+            	[
+            		'new_name' => 'no_extra_blank_lines',
+            		'note' => []
+            	],
+                'no_multiline_whitespace_before_semicolons' =>
+            	[
+            		'new_name' => 'multiline_whitespace_before_semicolons',
+            		'note' => []
+            	],
+                'no_short_echo_tag' =>
+            	[
+            		'new_name' => 'echo_tag_syntax',
+            		'note' => ['format' => 'long']    
+            	],
+                'php_unit_ordered_covers' =>
+            	[
+            		'new_name' => 'phpdoc_order_by_value',
+            		'note' => ['annotations' => ['covers']] 
+            	],
+                'phpdoc_inline_tag' =>
+            	[
+            		'new_name' => 'general_phpdoc_tag_rename, phpdoc_inline_tag_normalizer and phpdoc_tag_type',
+            		'note' => []    
+            	],
+                'pre_increment' =>
+            	[
+            		'new_name' => 'increment_style',
+            		'note' => ['style' => 'pre']    
+            	],
+                'psr0' =>
+            	[
+            		'new_name' => 'psr_autoloading',
+            		'note' => ['dir' => 'x'],    
+            	],
+                'psr4' =>
+            	[
+            		'new_name' => 'psr_autoloading',
+            		'note' => ''    
+            	],
+                'silenced_deprecation_error' =>
```

Comment:
i see that whitespaces are messed up here

Reply:
Done too